client/src/main/java/com/vesoft/nebula/client/graph/data/ResultSet.java

```diff
@@ -290,6 +289,10 @@ public Record rowValues(int index) {
 
     @Override
     public String toString() {
+        // When error, print the raw data directly
+        if (!isSucceeded()) {
+            return response.toString();
```

Comment:
If print the raw response, some informations are unreadable like this：
```
ExecutionResponse (
  error_code : E_CONFIG_NOT_FOUND (E_CONFIG_NOT_FOUND),
  latency_in_us : 501,
  space_name : 74 65 73 74 31 31 31,
  error_msg : 53 65 6D 61 6E 74 69 63 45 72 72 6F 72 3A 20 43 6F 6C 75 6D 6E 20 63 6F 75 6E 74 20 64 6F 65 73 6E 27 74 20 6D 61 74 63 68 20 76 61 6C 75 65 20 63 6F 75 6E 74 2E
)
```

Reply:
This doesn't need to be wrapped, the user really needs to know something else, just use the interface, such as calling `getErrorMsg` directly with the error message